paimon-core/src/main/java/org/apache/paimon/table/source/AbstractDataTableScan.java

```diff
@@ -244,6 +245,15 @@ protected StartingScanner createStartingScanner(boolean isStreaming) {
             case FROM_FILE_CREATION_TIME:
                 Long fileCreationTimeMills = options.scanFileCreationTimeMills();
                 return new FileCreationTimeStartingScanner(snapshotManager, fileCreationTimeMills);
+            case FROM_CREATION_TIMESTAMP:
+                Long creationTimeMills = options.scanCreationTimeMills();
+                return new CreationTimestampStartingScanner(
```

Comment:
I think ContinuousFromSnapshotStartingScanner is not needed. Just use a method to create StartingScanner.
And just test the method that created StartingScanner is right.